### PR TITLE
fix for gender voice

### PR DESCRIPTION
**Pull request by pjaudiomv**

### CODE REVIEW

**Comment by pjaudiomv** [comment]
Screen Shot 2020-11-15 at 1 48 17 PM

**Comment by pjaudiomv** [comment]
Screen Shot 2020-11-15 at 1 48 52 PM

fixes these things

**Comment by pjaudiomv** [comment]
actually this doesnt even fix the error, the fall back needs to be `voice must be man,woman,male,female,alice`

**Comment by pjaudiomv** [comment]
defaulting to alice

**Comment by pjaudiomv** [comment]
There's a validation warning that this should fix, we default to an actual voice. This only occurs if no voice is set obvs

File: endpoints/_includes/functions.php [hunk 1]
```diff
@@ -689,7 +689,7 @@ function voice($current_language = null)
     if (has_setting($current_language . "_voice")) {
         return setting($current_language . "_voice");
     } else {
-        return setting('voice');
+        return setting('alice');
```
Comment (radius314) [inline endpoints/_includes/functions.php:692]: There is already a default https://github.com/bmlt-enabled/yap/blob/master/endpoints/_includes/functions.php#L73

**Review by radius314 — CHANGES_REQUESTED** [review]

File: endpoints/_includes/functions.php (hunk 1 above)
Comment (pjaudiomv) [reply endpoints/_includes/functions.php:692]: Shut the front door, this makes sense now

File: endpoints/gender-routing.php [hunk 2]
```diff
@@ -6,6 +6,6 @@
 ?>
 <Response>
     <Gather language="<?php echo setting('gather_language') ?>" hints="<?php echo setting('gather_hints') ?>" input="<?php echo getInputType() ?>" timeout="10" speechTimeout="auto" action="gender-routing-response.php?SearchType=<?php echo $_REQUEST['SearchType']?>" method="GET">
-        <Say voice="<?php echo ('voice'); ?>" language="<?php echo setting('language') ?>">Press 1 to speak to a man, Press 2 to speak to a woman.</Say>
+        <Say voice="<?php echo voice(); ?>" language="<?php echo setting('language') ?>">Press 1 to speak to a man, Press 2 to speak to a woman.</Say>
```
Comment (radius314) [inline endpoints/gender-routing.php:9]: This change is good.

File: endpoints/_includes/functions.php (hunk 1 above)
Comment (pjaudiomv) [reply endpoints/_includes/functions.php]: removed as unneeded

**Review by radius314 — APPROVED** [review]

**Pull request merged**